.env

```diff
@@ -4,3 +4,7 @@ POSTGRES_HOST=resources-postgres
 POSTGRES_DB=resources-psql
 FLASK_ENV=development
 FLASK_DEBUG=1
+SEARCH_USER=search_user
+SEARCH_KEY=search_key
+SEARCH_RESOURCE_INDEX=search_resource_index
```

Comment:
In the same way, the docs call this an `index name`, can we match that terminology please?

Reply:
Meaning I think it should be `INDEX_NAME=search_resource_index`